plugins/woocommerce/src/Admin/API/Themes.php

```diff
@@ -192,7 +192,7 @@ public function get_item_schema() {
 	 * @return array
 	 */
 	public function get_collection_params() {
-		$params['context']   = $this->get_context_param( array( 'default' => 'view' ) );
+		$params = [ 'context' => $this->get_context_param( array( 'default' => 'view' ) ) ];
```

Comment:
```suggestion
		$params = array( 'context' => $this->get_context_param( array( 'default' => 'view' ) ) );
```

In WP code style rules, the longform `array` syntax is preferred to the shorter `[]`. In this case, I think it also makes it easier to understand.

Reply:
@coreymckrill I've fixed all PHP Lint issues https://github.com/woocommerce/woocommerce/pull/43233/commits/2355e202e0bc9ee131082939c825ef3441594f4d